_posts/2014-01-08-get-started.md

```diff
@@ -26,6 +26,8 @@ order: 1
     # Create DB tables and apply migrations
     $ foreman start
     # run webserver and celery tasks using Django dev server
+    # You can access the cabot deployment at http://localhost:5001/
```

Comment:
It's not really a "deployment" - what about "you can access your dev instance at ..."?